spec/controllers/users_controller_spec.rb

```diff
@@ -9,98 +9,30 @@
   end
 
   describe 'POST create' do
-    context 'with valid personal info and valid card' do
-      let(:charge) { double(:charge, successful?: true) }
-      let(:alice) { Fabricate(:user) }
-      let(:invitation) { Fabricate(:invitation, inviter: alice,
-                              recipient_email: 'joe@example.com') }
-
-      before { expect(StripeWrapper::Charge).to receive(:create).and_return(charge) }
-      after { ActionMailer::Base.deliveries.clear }
-
-      it 'creates the user' do
-        post :create, user: Fabricate.attributes_for(:user)
-        expect(User.count).to eq(1)
-      end
-
+    context 'successful user sign up' do
       it 'redirects to the sign in page' do
+        result = double(:sign_up_result, successful?: true)
+        UserSignUp.any_instance.should_receive(:sign_up).and_return(result)
```

Comment:
`should` syntax is no longer recommended. The `expect` syntax equivalent to this is

```ruby
expect_any_instance_of(UserSignUp).to receive(:sign_up).and_return(result)
```